### ISSUE TITLE
Unable to get SQS notification on S3 object creation

### ISSUE THREAD

**Issue by mathieuruellan** [localstack/localstack#3097]
I'm using the 0.11.6 version

I'm creating an SQS, a S3 with a configuration file to link create event with SQS but nothing appears in the queue.

```
echo "hello" > AnyFile.txt

export AWS_ACCESS_KEY_ID=foobar
export AWS_SECRET_ACCESS_KEY=foobar
export AWS_DEFAULT_REGION=us-east-1

docker run -d \
	--rm -p 4566:4566 \	
	-e "SERVICES=s3,sqs,sns" \
	localstack/localstack:0.11.6

aws --endpoint-url=http://localhost:4566 s3 mb s3://TESTNOTIF
aws --endpoint-url=http://localhost:4566 sqs create-queue --queue-name TESTQUEUENOTIF
aws --endpoint-url=http://localhost:4566 s3api put-bucket-notification-configuration --bucket TESTNOTIF --notification-configuration file://configuration.json
aws --endpoint-url=http://localhost:4566 s3 cp AnyFile.txt s3://TESTNOTIF
aws --endpoint-url=http://localhost:4566 sqs receive-message --queue-url http://localhost:4566/000000000000/TESTQUEUENOTIF
```

configuration.json:
```
{
  	"QueueConfigurations": [
		{
				"QueueArn": "arn:aws:sqs:us-east-1:000000000000:TESTQUEUENOTIF",
				"Events": [
								"s3:ObjectCreated:*"
							]				
		}
	]
}
```

**Comment by whummer** [comment]
Thanks for reporting @mathieuruellan . This should be fixed in #3106 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks

**Comment by mathieuruellan** [comment]
Thank you @whummer. It's working on my side!
